src/FetchWorkerTool.worker.js

```diff
@@ -55,7 +55,7 @@ const onMessage = ({data: job}) => {
             return Promise.reject(result.status);
         })
         .then(buffer => complete.push({id: job.id, buffer}))
-        .catch(error => complete.push({id: job.id, error}))
+        .catch(error => complete.push({id: job.id, error: error.message || `Failed request: ${job.url}`}))
```

Comment:
```suggestion
        .catch(error => complete.push({id: job.id, error: (error && error.message) || `Failed request: ${job.url}`}))
```

Just in case we somehow reject `undefined` or `null` ¯\_(ツ)_/¯